.github/workflows/description_manager.yaml

```
@@ -0,0 +1,219 @@
+            echo "**${ALL_CHANGED_FILES_COUNT}** file(s) changed, triggering DESCRIPTION update:"
+            echo ""
+            for file in ${ALL_CHANGED_FILES}; do
+              echo "- \`${file}\`"
+            done
+          } >> "$GITHUB_STEP_SUMMARY"
+
+          echo "run_next_job=true" >> $GITHUB_OUTPUT
+
+      - name: No changes
+        if: steps.changes.outputs.any_changed != 'true'
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:2:30: Double quote to prevent globbing and word splitting

(shellcheck)